java-compiler-testing/src/test/java/io/github/ascopes/jct/tests/unit/filemanagers/JctFileManagerImplTest.java

```diff
@@ -0,0 +1,29 @@
+package io.github.ascopes.jct.tests.unit.filemanagers;
+
+import io.github.ascopes.jct.filemanagers.JctFileManager;
+import io.github.ascopes.jct.filemanagers.ModuleLocation;
+import io.github.ascopes.jct.filemanagers.impl.JctFileManagerImpl;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@DisplayName("JctFileManager Tests")
+public class JctFileManagerImplTest {
```

Comment:
we don't need to mark a Junit5 test class as public, can be left package-private :)